.changeset/lucky-shirts-joke.md

```
@@ -0,0 +1,5 @@
+---
+"assistant-ui": patch
+---
+
+fix: correctly pass component URLs to shadcn CLI
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)